winml/lib/Api/LearningModelBinding.cpp

```diff
@@ -21,6 +21,11 @@ LearningModelBinding::LearningModelBinding(
 }
 WINML_CATCH_ALL
 
+LearningModelBinding::~LearningModelBinding()
+{
+  Clear();
+}
+
```

Comment:
Why is this needed?

Reply:
not needed! I was using this earlier when I didn't have Ort::Allocator

---
In reply to: [358556189](https://github.com/microsoft/onnxruntime/pull/2653#discussion_r358556189) [](ancestors = 358556189)

Reply:
Actually this is needed to ensure that Ort::Allocator destructs after Ort::Value

---
In reply to: [358556189](https://github.com/microsoft/onnxruntime/pull/2653#discussion_r358556189) [](ancestors = 358556189)